pkg/authorization/registry/clusterpolicy/strategy.go

```diff
@@ -47,14 +47,12 @@ func (strategy) PrepareForUpdate(obj, old runtime.Object) {
 	_ = obj.(*authorizationapi.ClusterPolicy)
 }
 
-// Validate validates a new policy.
 func (strategy) Validate(ctx kapi.Context, obj runtime.Object) fielderrors.ValidationErrorList {
-	return validation.ValidateClusterPolicy(obj.(*authorizationapi.ClusterPolicy))
+	return validation.Validator.Validate(obj)
```

Comment:
> The overall method is fine (test all validators, big switch).  But the individual strategies need to call their own validation methods.

Doing that makes it possible to check "", ".", "..", but "contains /" is hard to do that way.  Was validation rejected because "/" is illegal or because you want exactly 17 characters?  Requiring an exact string (you only get a string) makes it more realistic to be right, but not guaranteed.  Do you consider that risk acceptable?